src/Mono.Android/Android.Runtime/JNIEnv.cs

```diff
@@ -247,8 +247,13 @@ static void ManualJavaObjectDispose (Java.Lang.Object obj)
 		static void Initialize ()
 		{
 			if (mono_unhandled_exception == null) {
+#if NETCOREAPP
```

Comment:
This feels "wasteful", in that we're using Reflection to create a delegate for a method which is directly convertible to the desired type *without* using Reflection.

Thus, I'd prefer:

```csharp
#if NETCOREAPP
				mono_unhandled_exception = monodroid_debugger_unhandled_exception;
#else   // NETCOREAPP
				var mono_UnhandledException = typeof (System.Diagnostics.Debugger)
					.GetMethod ("Mono_UnhandledException", BindingFlags.NonPublic | BindingFlags.Static);
				if (mono_UnhandledException != null)
					mono_unhandled_exception = (Action<Exception>) Delegate.CreateDelegate (typeof(Action<Exception>), mono_UnhandledException);
#endif  // NETCOREAPP
```

Reply:
Fixed :)